### DISCUSSION TITLE
Welcome to Ollama_Agents Discussions!

### DISCUSSION THREAD

**Discussion by MikeyBeez** [MikeyBeez/Ollama_Agents#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GoldenGarage** [comment]
Hi.

I just downloaded, setup and ran Ollama_Agents.

I am getting the following error:
`Unexpected error: no such table: edges`

Is there a database initialization scrip that I need to run?

Thanks for your help!

**Comment by ipdro** [comment]
Hi, downloading the requirements, get an "ERROR: Could not find a version that satisfies the requirement duckduckgo_search==6.2.11" - 
that package have been renamed to ddgs. (their page on pypi.org says " _This package (duckduckgo_search) has been renamed to ddgs! Use pip install ddgs instead_".).
Please update the code :) thanks

**Reply by MirkoHuth** [reply]
6.2.11 does not exist. Change in requirements.txt to version 6.2.11b1 or higher

**Comment by MirkoHuth** [comment]
When applying requirements it stop when trying to install whisper:

Collecting openai-whisper==20231117 (from -r requirements.txt (line 79))
  Downloading openai-whisper-20231117.tar.gz (798 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 798.6/798.6 kB 11.4 MB/s  0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
